.travis.yml

```diff
@@ -135,7 +137,14 @@ install:
 script:
   - echo ${JAVA_HOME}
   - ${JAVA_HOME}/bin/java -version
-  - mx --primary-suite-path ${TRAVIS_BUILD_DIR}/${PRIMARY} --J @"-Xmx2g" --java-home=${JAVA_HOME} gate --strict-mode --tags ${GATE}
+  - |
+      if [[ ${RM_GIT} ]]
+      then
+        rm -rf .git
+        mx --primary-suite-path ${TRAVIS_BUILD_DIR}/substratevm native-image --version
```

Comment:
Why not run the same testing as in the `else` branch?
Also, I would rename `RM_GIT` to `WITHOUT_VCS` as it better documents the intention of this case.

Reply:
> Why not run the same testing as in the `else` branch?

That's a good question and kind of show why this kind of tests might not be that future proof.
Looking at the changes in https://github.com/oracle/graal/pull/2738/commits/05d53b05ae15cab2e6831161ae6a77b7e7213a5d one sees that only 3 suites are affected (sdk, compiler, substratevm).
Now in these suites only a subset of the commands are affected. For instance in substratevm the only affected commmands are `native-image` and `native-image-configure` (the ones invoking `_ensure_vm_built`. As a result, 
`mx --primary-suite-path substratevm gate --tags build` works fine even without this PR.

In order to test all the changes of this PR one needs to run:
1. `mx --primary-suite-path substratevm native-image -version`
2. `mx --primary-suite-path compiler makegraaljdk /tmp/graaljdk`
3. `mx --primary-suite-path sdk javadoc`

but this still allows people to perform changes that could break mx commands performed outside a vcs repository.
E.g.  `mx --primary-suite-path substratevm build`

Unfortunately we cannot (it would take too long) test every mx command of graal, so I just settled with `mx native-image`.
WDYT?

> Also, I would rename `RM_GIT` to `WITHOUT_VCS` as it better documents the intention of this change.

:+1: I'll take care of that.

Reply:
Ok, then we'll start with the testing you've added and expand as necessary (i.e. when we find other commands that need to be fixed to work without VCS).

Reply:
Thank u

Reply:
Thanks @dougxc!

Reply:
Have you looked into the Travis failures on this PR?

Reply:
Travis looks happy now :)